tap_eloqua/client.py

```diff
@@ -40,7 +48,21 @@ def __exit__(self, type, value, traceback):
                           max_tries=5,
                           factor=2)
     def get_access_token(self):
-        if self.__access_token is not None and self.__expires > datetime.utcnow():
+        if self.dev_mode:
+            config = read_config(self.__config_path)
+            try:
+                self.__access_token = config['access_token']
+                self.__refresh_token = config['refresh_token']
+                self.__expires=strptime_to_utc(config['expires_in'])
+            except KeyError as _:
```

Comment:
This should be a legitimate variable name. The `_` (or `_foo`/`_*`) name is a convention that indicates the variable is unused, but this one is used. In any case, if the value was unused, just doing `except KeyError:` would be the right way to do it.

Using `ex` for an exception name is generally what we do

Reply:
fixed